### DISCUSSION TITLE
`tags.type`: "project" vs "solution"

### DISCUSSION THREAD

**Discussion by mrlacey** [dotnet/templating#4771] [Template authoring] [answered]
In a `template.json` file the `tags` collection is defined as requiring a `type` property that can be either "item", "project", or "solution".

I have a template that creates a solution with multiple projects in it and the template includes a `.sln` file.

This seems to work fine when I set the type as either "project" or "solution". 

I can't find any documentation on the differences between specifying the type as "project" or "solution."
Is there one?
When should I use one or the other?

**Comment by vlada-shubina** [comment, marked as answer]
`solution` type makes difference only when used in Visual Studio. https://aka.ms/netcore-templates explains the details. 
I would recommend to use `solution` type anyway if your template is producing `sln` for better categorization even in CLI context.

**Reply by mrlacey** [reply]
thank you for clarifying
